ui/public/metadata.json

```diff
@@ -12,7 +12,8 @@
   ],
   "docs": {
     "documentation_url": "https://github.com/nethesis/ns8-nethvoice-proxy",
-    "code_url": "https://github.com/nethesis/ns8-nethvoice-proxy"
+    "code_url": "https://github.com/nethesis/ns8-nethvoice-proxy",
+    "bug_url": "https://github.com/nethesis/ns8-nethvoice/issues"
```

Comment:
Why did you add the `bug_url` field?

Reply:
Because it was missing, in the UI there is a field that shows it but it was missing.

![image](https://github.com/nethesis/ns8-nethvoice-proxy/assets/6152486/77034948-dd04-4634-8b75-152e6861de52)

Reply:
So is the `bug_url` field mandatory?

Reply:
Yes, otherwise you have a blank line near the "Bugs" field

Reply:
Ok, then the correct value is:
```suggestion
    "bug_url": "https://github.com/NethServer/dev"
```